short-response.md

```diff
@@ -30,12 +43,12 @@
 
   console.log(status);
   ```
-
+  - No error here because `status` is declared with `var`, meaning it has function scope and being inside the `if` block does not make it inaccesible to the `console.log();`.
 
 
 4. **In JavaScript, we can declare variables with `var`, `let`, and `const`. What are the differences between each? Be sure to comment on how each declaration impacts the _scope_, _reassignment_, and _hoisting_ of variables.**
 
-
+  - Variables declared with `let` and `const` have block scope, and are uninitialized when hoisted. `const` variables cannot be reassiged. `var` variables have function scope and are hoisted as `undefined`.
```

Comment:
# Question 4 Score:
## Technical
**3**: Perfect

## Writing
**3**: Clear and concise